snap/snapcraft.yaml

```diff
@@ -22,14 +22,24 @@
           grade:                  devel
           icon:                   images/logo.svg
           type:                   app
-
           apps:
             sqlitebrowser:
               command:            desktop-launch $SNAP/bin/sqlitebrowser
               desktop:            share/applications/sqlitebrowser.desktop
               environment:
                 DISABLE_WAYLAND:  1
                 TMPDIR:           $XDG_RUNTIME_DIR
+                XLOCALEDIR: '$SNAP/usr/share/X11/locale'
+                LOCPATH: '$SNAP/usr/lib/locale'
+                GTK_PATH: $SNAP/lib/gtk-2.0
```

Comment:
Should this have quote characters around the string, like in `LOCPATH` above?

Same question seems to apply to several of the lines. :wink:

Reply:
i tested the changes, that is weird, after putting these vars in string, QT theme doesn't work. It falls back to GTK look and feel.

Reply:
No worries, I just wasn't sure. :smile: